common/test/acceptance/tests/lms/test_lms_course_home.py

```diff
@@ -133,3 +133,14 @@ def test_course_home_a11y(self):
         course_home_page = CourseHomePage(self.browser, self.course_id)
         course_home_page.visit()
         course_home_page.a11y_audit.check_for_accessibility_errors()
+
+    def test_course_search_a11y(self):
```

Comment:
This needs the decorator ``@attr('a11y')`` to ensure that this runs as an a11y test and not a regular Bok Choy test.